client/index.html

```diff
@@ -1,11 +1,473 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <template data-type="pug" data-src="./view/base/meta.pug" />
-    <link rel="stylesheet" href="./index.css">
-    <script type="module" src="./index.js" defer></script>
-  </head>
-  <body>
-    <template data-type="pug" data-src="./view/browsers/browsers.pug" />
-  </body>
+<head>
+  <link rel="stylesheet" href="./index.css">
+  <script type="module" src="./index.js" defer></script>
+  <meta name="color-scheme" content="dark light">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+  <article class="Article Article--intro">
+      <div class="Article--intro__header">
+        <h1 class="Article__h1">
+          <img src="./assets/browserlist_logo.svg" role="presentation">
+          Browserslist
+        </h1>
+        <a class="Badge" href="https://github.com/browserslist/browserslist">
+          <img src="./assets/github_logo.svg" role="presentation">
+          10.5k
+        </a>
+        <a class="Badge" href="https://twitter.com/Browserslist">
+          <img src="./assets/twitter_logo.svg" role="presentation"> 4k
+        </a>
+      </div>
+      <p class="Article--intro__about">Shared browser and platform compatibility config for popular JavaScript tools,
+        including
+        Autoprefixer, Babel, ESLint, PostCSS, and Webpack</p>
+      <p class="Article--intro__supported">
+        Supported by
+        <img src="./assets/evil_maritians_logo.svg" role="presentation">
+        <a href="" class="Link">Evil Martians</a> and
+        <img src="./assets/cube_logo.svg" role="presentation">
+        <a href="" class="Link">Cube</a>
+      </p>
+  </article>
+
+  <article class="Article Article--stats">
+    <h2>Display compatible browsers</h2>
+    <form id="query_form" class="Form">
+      <textarea name="query" cols="30" rows="6" required
+                class="Form__queryTextArea" id="query_text_area"
+                placeholder="Write the Browserslist config here..."></textarea>
+      <p class="Form__interactivePlaceholder">Write the Browserslist config here... <br>
+        For example
+        <a class="QueryLink" data-query="defaults"><code>defaults</code></a> ,
+        <a class="QueryLink" data-query="not dead"><code>not dead</code></a>
+      </p>
+      <p class="Form__hint Form__hint--progress">Press Enter
+        <span role="presentation" class="Form__enterIcon">
+          <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="none">
+            <path
+              d="M10.71 1.42h-.8a.1.1 0 0 0-.1.1v6.94H3.28V7.5a.1.1 0 0 0-.17-.09l-1.9 1.5a.1.1 0 0 0 0 .17l1.9 1.5a.1.1 0 0 0 .17-.08v-1h6.67c.48 0 .86-.4.86-.87v-7.1a.1.1 0 0 0-.1-.1Z"
+              fill="#1E1E1E" fill-opacity=".4"/>
+          </svg>
+        </span>
+        to submit
+      </p>
+      <p class="Form__hint Form__hint--required">Query should be not empty</p>
+      <p class="Form__hint Form__hint--error" id="error_message">Unknown query</p>
```

Comment:
ID without block prefix could lead to make conflict. Let's find good strategy for HTML-to-JS communication which will work with many cases (not only unique elements) and will have block isolation

Reply:
Let's take `[data-attributes]`. They are customizable and may not be unique